### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/911e90936dca3513a3d9c376ce7b095049c17b5e

### CODE REVIEW

**Review by peiying2 — COMMENTED** [review]
@stephaniehobson Overall looks good. I have a comment on one string that could be a concatenation issue.

File: en/firefox/products.ftl [hunk 1]
```diff
@@ -4,63 +4,73 @@
...
+firefox-products-by-mozilla = by { -brand-name-mozilla }
 
-# Obsolete string
-# The strong tag wraps a word that gets highlighted with a colorful underline for emphasis. The underline breaks if it is on two words, please omit the strong tags if they need to be around multiple words in your language
-firefox-products-firefox-is-more-than-a-browser-emphasis = { -brand-name-firefox } is <strong>more</strong> than a browser
+## Firefox
 
-firefox-products-mozilla-monitor = { -brand-name-mozilla-monitor }
+firefox-products-firefox = { -brand-name-firefox }
+firefox-products-get-the-browsers-that-block = Get the browsers that block 2000+ data trackers automatically. Enhanced Tracking Protection comes standard in every { -brand-name-firefox } browser.
```
Comment (flodolo) [inline en/firefox/products.ftl:27]: Why "browsers"? That seems wrong, given it points to the download page for Firefox desktop release.
https://www-dev.allizom.org/en-US/products/

File: en/firefox/products.ftl [hunk 2]
```diff
@@ -4,63 +4,73 @@
...
-firefox-products-relay = { -brand-name-firefox-relay}
-firefox-products-protect-your-real = Protect your real email address to help control your inbox.
-firefox-products-learn-more-about-relay = Learn more about { -brand-name-firefox-relay }
 
-firefox-products-send-your-large-files-and = Send your large files and sensitive documents safely, up to 2.5G.
-firefox-products-send-a-file = Send a file
+## MDN Plus
+
+firefox-products-mdn-plus = { -brand-name-mdn-plus }
+firefox-products-resources-for-developers = Resources for developers, by developers. Support MDN and make it your own with collections, notifications, and playgrounds.
```
Comment (flodolo) [inline en/firefox/products.ftl:55]: `MDN` -> `{ -brand-name-mdn }`

File: en/firefox/products.ftl [hunk 3]
```diff
@@ -4,63 +4,73 @@
...
 firefox-products-firefox-beginning = { -brand-name-firefox } is just the beginning
 
-# Obsolete string
-# HTML page description
-firefox-products-its-a-whole-family-of-products = It’s a whole family of products designed to keep you safer and smarter online.
+firefox-products-mozillas-family-of-products-sentence = { -brand-name-mozilla }’s family of products are all designed to keep you safer and smarter online.
 
-firefox-products-mozillas-family-of-products = { -brand-name-mozilla }’s family of products are all designed to keep you safer and smarter online
+# appears after the name of a product with independant branding. Example: Fakespot by Mozilla
+firefox-products-by-mozilla = by { -brand-name-mozilla }
```
[the remark this answers is not in the document]
Comment (alexgibson) [reply en/firefox/products.ftl:22]: I think for now we are probably OK to lose the "by Mozilla" as this language will be figured out properly in the redesign. Instead I'll add "Made by Mozilla" to the description for each.

**Comment by alexgibson** [comment]
@flodolo I pushed through a change that should hopefully rectify the comments here, thanks

**Review by flodolo — APPROVED** [review]
Thanks, looks good to me now.

**Comment by stephaniehobson** [comment]
@alexgibson Thank you for handling this while I was out sick.

File: en/firefox/products.ftl [hunk 4]
```diff
@@ -4,63 +4,70 @@
...
+firefox-products-fakespot = { -brand-name-fakespot }
+# scam artists are people who are trying to sell fakes or steal money
+# 'weeding' is a gardening term, here it is used to mean "removing unwanted things without touching the good ones"
+firefox-products-fakespot-has-your = { -brand-name-fakespot } has your back when you’re shopping online — weeding out scam artists and finding you the best deals. Made by { -brand-name-mozilla }.
+firefox-products-analyze = Analyze a URL
+
+## Pocket
+
+firefox-products-pocket = { -brand-name-pocket }
+firefox-products-discover-the-best-content = Discover the best content on the web — and consume it wherever and whenever you want. Made by { -brand-name-mozilla }.
```
Comment (flodolo) [inline en/firefox/products.ftl:72]: GitHub's diff structure made me miss that this is modifying an existing string :-( 

Which means we now have a ton of errors across locales.

Opened https://github.com/mozilla/bedrock/pull/14555

**Pull request merged**